docs/release.md

```diff
@@ -23,6 +23,15 @@ git push origin $REL_VER
 Or if you want to build/push images for local development.
 
 ```sh
+# Set the container registry for the images, for example:
+export CONTAINER_REGISTRY=us-docker.pkg.dev/my-project/images
+
+# Set the JVS docker image tag, for example:
+export JVS_DOCKER_TAG="0.0.5"
```

Comment:
Just so you know this image won't work. We need a new JVS release to get the plugin code work.

Reply:
Ah, yeah, just using this version here to test it locally. Will update it to the real version after we release a new JVS version

Reply:
Per offline conversation, JVS_DOCKER_TAG will be hard-coded in goreleaser.docker.yaml file, the base image change will be captured as PR so it is possible to infer overall binary change in the docker images between releases. Updated to hard-code now